### PR TITLE
added tooltip menu and clear theme

### PR DESCRIPTION
**Pull request by esbenjuul**

### CODE REVIEW

**Comment by gavmck** [comment]
This is pretty neat! Is it possible to make it show the selected theme in the toolbar so we can see which is loaded at a glance?

**Comment by esbenjuul** [comment]
Good idea! I have updated the PR.

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "ts3.5/**/*"
   ],
   "scripts": {
-    "prepare": "babel --extensions .ts,.tsx ./src --out-dir ./dist",
+    "prepare": "babel --extensions .ts,.tsx ./src --watch --out-dir ./dist",
```
[the remark this answers is not in the document]
Comment (esbenjuul) [reply package.json]: You're so right it was dev leftover. Now with new gif and removed watch flag!

**Comment by gavmck** [comment]
Awesome! I'm going to merge this and just add a bit to show "no theme selected" so it's clear what the button is for. Thank you!

**Pull request merged**